### ISSUE TITLE
Issue with compacting path names using odgi -viz

### ISSUE THREAD

**Issue by Gavin-I** [pangenome/odgi#527]
Hello,

I am currently using odgi -viz to produce graphs and I don't understand how to compact my graph path names.

Currently when I run it through cactus-pangenome, the graph only has the names of the supplied input file. However, when I run an odgi file without using cactus-pangenome, and using odgi -viz directly (as I want to use different options), there are many more path names within the graph, making my graph really big on the vertical scale. I tried to use -M option to see if it was the correct option to compact the graph, however, the graph was not changed.

I was wondering if there was a tutorial or example of how to compact the graph to only those of the input file.

Thank you for any help!

**Comment by subwaystation** [comment]
Hi @Gavin-I,

could you please be more specific? Do you have some example command lines?
What do you mean with "odgi file without cactus-pangenome"?

**Comment by Gavin-I** [comment]
Hello, thank you both for the help.

To be more specific, currently I am creating graphs of my data by using cactus-pangenome.
 The command looks something like this (From their basic example):

cactus-pangenome ./js ./examples/evolverPrimates.txt --outDir primates-pg --outName primates-pg --reference simChimp --viz

In which the option --viz generates a graph for me. 

I wanted to recreate the graph directly using ODGI and I used the command:
 odgi viz -i  example.og  -o  example.png  -x 500

 I couldn't get the graphs to look similar, as I could not get the graph to contain less path names.


However, thank you @AndreaGuarracino , afterwards I tried using your command and it seems to be doing what I wanted.

The graphs look very similar although, with some minor differences which might be because of some additional options I left out.



Once again, thank you both for the help!
